server/pbenchinacan/pod-nodata.yml

```diff
@@ -1,11 +1,17 @@
 #
 # This configuration is intended to create a brand new (empty) Pbench Server
-# deployment.  The resulting pod consists of three containers: the Pbench Server
-# and its associated Elasticsearch and PostgreSQL database servers.
+# deployment.  The resulting pod consists of four containers: the Pbench Server
+# and its associated Elasticsearch, PostgreSQL database servers and a Keycloak
+# authentication server.
 #
-# The Elasticsearch and PostgreSQL container images have specific versions and
-# are pulled from cached images in an internal registry; the Pbench Server
-# container image is pulled from the same registry with a configurable tag.
+# Note: Even though this pod is for creating empty containers, the Keycloak
+# container does come with some basic pre-configuration like a realm and client
+# that pbench-server expects to be there before the startup.
```

Comment:
I wouldn't say "empty containers". The distinction is whether we have pre-loaded Pbench datasets. We need to mimic the real production configuration with our backend databases, including our Keycloak broker. And I've always thought we were going to need to create an `admin` user. (Note that this is essential for auditing as well as the server configuration API.)